pkg/util/env.go

```diff
@@ -114,33 +120,63 @@ func GetElasticsearchDataNodeRequestMemory() string {
 	return GetEnvFunc(esDataNodeRequestMemory)
 }
 
+// GetElasticsearchDataStorageSize returns the Elasticsearch storage size request
+func GetElasticsearchDataStorageSize() string {
+	return GetEnvFunc(esDataStorageSize)
+}
+
 // GetGrafanaRequestMemory returns the Grafana memory request resource.
 func GetGrafanaRequestMemory() string {
 	return GetEnvFunc(grafanaRequestMemory)
 }
 
+// GetGrafanaDataStorageSize returns the Prometheus storage size request
+func GetGrafanaDataStorageSize() string {
+	return GetEnvFunc(grafanaDataStorageSize)
+}
+
 // GetPrometheusRequestMemory returns the Prometheus memory request resource.
 func GetPrometheusRequestMemory() string {
 	return GetEnvFunc(prometheusRequestMemory)
 }
 
+// GetPrometheusDataStorageSize returns the Prometheus storage size request
+func GetPrometheusDataStorageSize() string {
+	return GetEnvFunc(prometheusDataStorageSize)
+}
+
 // GetKibanaRequestMemory returns the Kibana memory request resource.
 func GetKibanaRequestMemory() string {
 	return GetEnvFunc(kibanaRequestMemory)
 }
 
-// GetElasticsearchMasterNodeReplicas returns the Elasticsearch master replicas.
-func GetElasticsearchMasterNodeReplicas() int32 {
-	value := GetEnvFunc(esMasterNodeReplicas)
+// getEnvReplicaCount gets a replica count from the named env var, returning the provided default if not present
+func getEnvReplicaCount(envVarName string, defaultValue int32) int32 {
+	value := GetEnvFunc(envVarName)
 	if len(value) != 0 {
 		count, err := strconv.ParseInt(value, 10, 32)
 		if err != nil {
-			zap.S().Infof("%v is invalid.  The default value of 3 will be used as replicas of ES master node.", value)
+			zap.S().Infof("%s: %v is invalid.  The default value of %v will be used for node replicas.", envVarName, value, defaultValue)
 		} else {
 			return int32(count)
 		}
 	}
-	return 3
+	return defaultValue
+}
+
+// GetElasticsearchMasterNodeReplicas returns the Elasticsearch master node replicas.
+func GetElasticsearchMasterNodeReplicas() int32 {
+	return getEnvReplicaCount(esMasterNodeReplicas, 3)
```

Comment:
should we have constants defined for the default values?

Reply:
can't hurt, I'll make that change.

Reply:
was about to say not a blocking comment

Reply:
Pushed an update to use constants